components/standings/standings_storage.lua

```diff
@@ -12,10 +12,12 @@ local Variables = require('Module:Variables')
 local StandingsStorage = {}
 
 function StandingsStorage.run(index, data)
+	local title = data.title or ''
+	local cleanedTitle = title:gsub('<.->.-</.->', '')
```

Comment:
Could limit it to just spans for now. 
```suggestion
	local cleanedTitle = title:gsub('<span.->.-</span>', '')
```

Reply:
The problem with only allowing spans is that the app will break if an edge case slips through. It would be better to err on the side of cutting too much, and then adjusting later.

Reply:
okay for me